Description: Mention smbldap-tools package in examples/LDAP/README
Author: Christian Perrier <bubulle@debian.org>
Bug-Debian: http://bugs.debian.org/341934
Forwarded: not-needed

Index: samba-debian-pkg/examples/LDAP/README
===================================================================
--- samba-debian-pkg.orig/examples/LDAP/README	2015-10-07 04:59:20.000000000 -0700
+++ samba-debian-pkg/examples/LDAP/README	2015-10-07 04:59:20.095575746 -0700
@@ -72,6 +72,9 @@
 On Debian systems, the smbldap-tools exists as a separate package
 and is not included in LDAP examples.
 
+On Debian systems, the smbldap-tools exists as a separate package
+and is not included in LDAP examples.
+
 !==
 !== end of README
 !==
